### ISSUE TITLE
Classic Story Zone failure

### ISSUE THREAD

**Issue by CodeMonter** [TheGameCreators/GameGuruRepo#4243]
I tested the Story Zone with a simple map (Start marker, Story Zone & a decal to mark its place). I pointed the Story Zone at one of the provided sample videos.
I tested it within the editor (Test Game) and as a Standalone game.
In both tests, Game Guru simply froze. Everything stopped moving. Nothing displayed.
In the editor, Esc did nothing. Tapping the Windows key got GG to reload and "Recover Previous".
When I tapped the Windows key in the standalone, it crashed, removing my mouse pointer. Several failure messages displayed as I tapped the Enter key on them. Eventually I was able to close the program.
System: Windows 7.

**Comment by Teabone3** [comment]
It works fine for me. What sample video is crashing on you?

**Comment by CodeMonter** [comment]
Interesting!  Thank you for commenting, Teabone3.
For me, everything I tried failed.
Since it works for you, this sounds like an issue with specific graphics chipsets or drivers.
I'll try exploring that idea.  I'm away from home for a few days.  When I return, I'll post my graphics card.

**Comment by CodeMonter** [comment]
OK, I'm back.  I just made an additional test.
1) I copied the standalone program to a Windows 10 laptop with a very different graphics chipset & driver.  It works perfectly.
2) My graphics card on my Game Guru Windows 7 desktop machine is an NVIDIA GeForce GT630.  An attempt to update the driver tells me it's already the latest version.

Sooo... this looks like something about the GT630 is incompatible with Game Guru's Story Zone videos.

Since it works for Teabone and on my W10 laptop, this may be too rare a problem to justify the resources to fix it.  If I want to use Story Zones in Game Guru, I'll have to test them on the other machine.
